### PR TITLE
Roman ramp fitting docs (RCAL-130)

### PR DESCRIPTION
**Pull request by dmggh**
Closes #

Resolves RCAL-130

**Description**

This PR addresses RCAL-130: documentation for Roman ramp fitting

Checklist
- [ ] Tests

- [ X] Documentation

- [ ] Change log

- [ ] Milestone

- [ ] Label(s)

### CODE REVIEW

File: docs/roman/ramp_fitting/description.rst [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+Segment-specific Computations:
+------------------------------
+The variance of the slope of a segment due to read noise is:
+
+.. math::  
+   var^R_{s} = \frac{12 \ R^2 }{ (ngroups_{s}^3 - ngroups_{s})(tgroup^2) } \,,
+
+where :math:`R` is the noise in the difference between 2 frames, 
+:math:`ngroups_{s}` is the number of groups in the segment, and :math:`tgroup` is the group 
+time in seconds (from the keyword TGROUP).
```
Comment (ddavis-stsci) [inline docs/roman/ramp_fitting/description.rst:164]: I don't think we have the keyword TGROUP in the asdf file? At least I don't see it in RAD.

File: docs/roman/ramp_fitting/description.rst [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+The square root of the combined variance is stored in the ERR array of the primary output.
+
+The overall slope depends on the slope and the combined variance of the slope of all
+segments, so is a sum over segments:
+
+.. math::    
+    slope_{o} = \frac{ \sum_{s}{ \frac{slope_{s}} {var^C_{s}}}} { \sum_{s}{ \frac{1} {var^C_{s}}}}
+
+
+Upon successful completion of this step, the status keyword S_RAMP will be set
```
Comment (ddavis-stsci) [inline docs/roman/ramp_fitting/description.rst:214]: keyword ---> attribute and this should set cal_step? or do we need another attribute?

File: docs/roman/ramp_fitting/description.rst [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+Error Propagation
+=================
+
+Error propagation in the ramp fitting step is implemented by storing the
+square-root of the exposure-level combined variance in the ERR array of the primary
+output product. This combined variance of the exposure-level slope is the sum
+of the variance of the slope due to the Poisson noise and the variance of the 
+slope due to the read noise. These two variances are also separately written
+to the extensions VAR_POISSON and VAR_RNOISE in the primary output.
```
Comment (ddavis-stsci) [inline docs/roman/ramp_fitting/description.rst:226]: Should extensions be arrays, 
... written to the arrays VAR_POISSON and VAR_RNOISE in the asdf file.

File: docs/roman/ramp_fitting/description.rst [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+For the optional output product, the variance of the slope due to the Poisson
+noise of the segment-specific slope is written to the VAR_POISSON extension.
+Similarly, the variance of the slope due to the read noise of the
+segment-specific slope  is written to the VAR_RNOISE extension.
```
Comment (ddavis-stsci) [inline docs/roman/ramp_fitting/description.rst:231]: VAR_RNOISE extension  --> VAR_RNOISE array?

**Review by ddavis-stsci — APPROVED** [review]
A few minor comments otherwise it looks good.

File: docs/roman/ramp_fitting/description.rst (hunk 1 above)
Comment (dmggh) [reply docs/roman/ramp_fitting/description.rst:164]: I do see group_time in exposure.

File: docs/roman/ramp_fitting/description.rst (hunk 2 above)
Comment (dmggh) [reply docs/roman/ramp_fitting/description.rst:214]: " Upon successful completion of this step, the status attribute ramp_fit will be set ..."

File: docs/roman/ramp_fitting/description.rst (hunk 3 above)
Comment (dmggh) [reply docs/roman/ramp_fitting/description.rst:226]: Yes, "arrays"

File: docs/roman/ramp_fitting/description.rst (hunk 4 above)
Comment (dmggh) [reply docs/roman/ramp_fitting/description.rst:231]: Yes

**Review by ddavis-stsci — APPROVED** [review]

**Pull request merged**